hitbox/src/lib.rs

```diff
@@ -1,155 +1,66 @@
-//! A proxy actor and infrastructure for asynchronous and clear cache interaction for Actix actor and Actix-web frameworks.
+//! An a implementation and infrastructure for asynchronous and clear cache integration.
 //!
-//! # A tour of hitbox
+//! # A quick tour of hitbox
 //!
-//! This crate consist of three main part:
-//! * [CacheActor] actix actor.
-//! * [Backend] trait and its implementation ([RedisBackend]).
+//! Our crates consist of next main part:
 //! * [Cacheable] trait.
+//! * [Backend] trait and its implementation ([RedisBackend]).
```

Comment:
* [Backend] trait and its implementation (For now the implementation for [RedisBackend] is ready).